usaspending_api/etl/management/commands/update_file_c_linkages.py

```diff
@@ -26,3 +31,28 @@ def handle(self, *args, **options):
     def run_sql(self, submission=None):
         for link_type in self.LINKAGE_TYPES:
             update_c_to_d_linkages(type=link_type, submission_id=submission)
+
+    def faba_award_id_check(self):
+        """Update any FABA records with an award_id associated with an award
+        that no longer exists."""
```

Comment:
Can you update some of these comments to include the following language? 
> Unlinks FABA records from Awards that no longer exist

Same for the comment in `usaspending_api/etl/management/sql/c_file_linkage/update_faba_award_ids.sql`

Reply:
Updated the comments in commits 75eeaae3e03db6ad7a4f5864d7c9611f811922e4 and 3b7e9b731d84ce83a02902478eb03a22a5e7e2b4